ResearchKit/Common/ORKSkin.m

```diff
@@ -271,7 +277,8 @@ CGFloat ORKStandardHorizontalMarginForWindow(UIWindow *window) {
         default:
             margin = ORKStandardLeftTableViewCellMarginForWindow(window);
             break;
-        case ORKScreenTypeiPad:{
+        case ORKScreenTypeiPad:
+        case ORKScreenTypeiPad12_9:{
```

Comment:
We might need to do separate calculation for iPad and iPad12.9, because `ORKiPadScreenSize.width` was used in the calculation.
In the calculation for iPad12.9,  `ORKiPad12_9ScreenSize` should be used.